addons/skin.estouchy/language/resource.language.en_gb/strings.po

```diff
@@ -386,7 +386,51 @@ msgctxt "#31565"
 msgid "Show deleted"
 msgstr ""
 
-#empty strings from id 31566 to 31899
+#empty strings from id 31566 to 31744
+
+#. Window screen calibration: calibration from the rectangle box
+#: xml/SettingsScreenCalibration.xml
+msgctxt "#31745"
+msgid "[B]Pixel ratio adjustment[/B][CR]Adjust the rectangle so it is perfectly square"
```

Comment:
I don't think it's a good practice to mix styling with translations. Although in addons that are 100% under your control this might be fine, in generic code I'm tempted to say we should avoid that.
Why not use the bold and carriage return tags within the skin xml files itself instead of forcing them right away on the gettext files?

Reply:
the text is set by the source code not in the xml, but i think i can split all sentences in to two ID's, then formatting it from code

Reply:
removed style in the translations